blockchain/proxy_test.go

```diff
@@ -1203,3 +1241,231 @@ func TestProxy_IsDataTrieMigrated(t *testing.T) {
qqqqqqqqqqqpgqta0tv8d5pjzmwzshrtw62n4nww9kxtl278ssspxpxu","gasPrice":1000000000,"data":"RVNEVFRyYW5zZmVyQDQ4NTU1NDRiMmQzNDY2NjEzNDYyMzJANThhODZmYzg0MzQw","previousTransactionHash":"be72a85d881e83a22196715a54f5897f40210c1e62162014f47ab1be89e7702c","originalTransactionHash":"be72a85d881e83a22196715a54f5897f40210c1e62162014f47ab1be89e7702c","originalSender":"erd1mmh2j5y8esv2tmmyeau3hr4xa2u2te3zc3j9wumn3v5vm8uvsczqnucj5l","sourceShard":1,"destinationShard":0,"miniblockType":"SmartContractResultBlock","miniblockHash":"13b5b686e27dca2c3cbd131837ba8050639970661717a4fef8bde968f738f766","logs":{"address":"erd1mmh2j5y8esv2tmmyeau3hr4xa2u2te3zc3j9wumn3v5vm8uvsczqnucj5l","events":[{"address":"erd1qqqqqqqqqqqqqpgqta0tv8d5pjzmwzshrtw62n4nww9kxtl278ssspxpxu","identifier":"ESDTTransfer","topics":["SFVUSy00ZmE0YjI=","","WKhvyENA","3u6pUIfMGKXvZM95G46m6ril5iLEZFdzc4sozZ+MhgQ="],"data":null,"additionalData":["","RVNEVFRyYW5zZmVy","SFVUSy00ZmE0YjI=","WKhvyENA"]},{"address":"erd1mmh2j5y8esv2tmmyeau3hr4xa2u2te3zc3j9wumn3v5vm8uvsczqnucj5l","identifier":"writeLog","topics":["AAAAAAAAAAAFAF9eth20DIW3Chca3aVOs3OLYy/q8eE="],"data":"QDZmNmI=","additionalData":["QDZmNmI="]},{"address":"erd1mmh2j5y8esv2tmmyeau3hr4xa2u2te3zc3j9wumn3v5vm8uvsczqnucj5l","identifier":"completedTxEvent","topics":["vnKoXYgeg6IhlnFaVPWJf0AhDB5iFiAU9HqxvonncCw="],"data":null,"additionalData":null}]},"status":"success","tokens":["HUTK-4fa4b2"],"esdtValues":["97480453145408"],"operation":"ESDTTransfer","callType":"directCall","options":0},{"type":"unsigned","processingTypeOnSource":"MoveBalance","processingTypeOnDestination":"MoveBalance","hash":"43a29c450a306a8e82c40f3a453892a8827d986367b95d5e7768f35e1b582c26","nonce":30335,"round":21008275,"epoch":1458,"value":"77706790000000","receiver":"erd1mmh2j5y8esv2tmmyeau3hr4xa2u2te3zc3j9wumn3v5vm8uvsczqnucj5l","sender":"erd1qqqqqqqqqqqqqpgqta0tv8d5pjzmwzshrtw62n4nww9kxtl278ssspxpxu","gasPrice":1000000000,"data":"QDZmNmJAMDAwMDAwMGI0ODU1NTQ0YjJkMzQ2NjYxMzQ2MjMyMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDA2NThhODZmYzg0MzQw","previousTransactionHash":"be72a85d881e83a22196715a54f5897f40210c1e62162014f47ab1be89e7702c","originalTransactionHash":"be72a85d881e83a22196715a54f5897f40210c1e62162014f47ab1be89e7702c","sourceShard":1,"destinationShard":0,"miniblockType":"SmartContractResultBlock","miniblockHash":"13b5b686e27dca2c3cbd131837ba8050639970661717a4fef8bde968f738f766","logs":{"address":"erd1mmh2j5y8esv2tmmyeau3hr4xa2u2te3zc3j9wumn3v5vm8uvsczqnucj5l","events":[{"address":"erd1mmh2j5y8esv2tmmyeau3hr4xa2u2te3zc3j9wumn3v5vm8uvsczqnucj5l","identifier":"completedTxEvent","topics":["vnKoXYgeg6IhlnFaVPWJf0AhDB5iFiAU9HqxvonncCw="],"data":null,"additionalData":null}]},"status":"success","operation":"transfer","isRefund":true,"callType":"directCall","options":0}],"indexOfFirstTxProcessed":0,"indexOfLastTxProcessed":1}],"scheduledData":{"rootHash":"a0f86f73d739d8490d07d3a47acdd19a14c80ecfe8247aca2877d4e06944cc8f","accumulatedFees":"0","developerFees":"0"}}},"error":"","code":"successful"}`
```

Comment:
For better readability:
Format this string as an indented JSON and save it in separate .json files located in `/blockchain/testdata` directory
The tests will be simplified by just loading some .json files. This will separate (large) data from the test code.